packages/yew/src/html/classes.rs

```diff
@@ -29,17 +32,21 @@ fn build_string<'a>(first: &'a str, rest: impl Iterator<Item = &'a str> + Clone)
             .sum(),
     );
 
-    s.push_str(first);
-    s.extend(rest.flat_map(|class| [" ", class]));
-    s
+    s.push_str(first.as_str());
+    // NOTE: this can be improved once Iterator::intersperse() becomes stable
+    for class in rest {
+        s.push_str(" ");
+        s.push_str(class.as_str());
+    }
```

Comment:
I had to change this because `class.as_str()` borrows on `class` and the compiler refused to yield the borrow value :disappointed: not a big deal